draft-ietf-rtcweb-jsep.xml

```diff
@@ -1097,6 +1097,54 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           simulcast as described in
           <xref target="sec.simulcast" />).</t>
         </section>
+
+        <section title="stopTransceiver" anchor="sec.transceiver-stop">
+          <t>
+            The stopTransceiver method stops an RtpTransceiver.
+            See <xref target="sec.transceiver-stopped"/> for more details.
+          </t>
+        </section>
+        <section title="isTransceiverStopped" anchor="sec.transceiver-stopped">
+          <t>
+            The isTransceiverStopped method returns true if
+            the transceiver has been stopped, either by a call
+            to stopTransceiver or by a call to
+            setRemoteDescription, and false otherwise.
+          </t>
+          <t>
+            A stopped RtpTransceiver does not send any
+            outgoing RTP or RTCP or process any incoming RTP
+            or RTCP.
+          </t>
+        </section>
+        <section title="setTransceiverDirection" anchor="sec.transceiver-set-direction">
+          <t>
+            The setTransceiverDirection method sets the
+            direction of a transceiver, which affects the
+            direction attribute of an associted m=section on
+            future calls to createOffer and createAnswer.
+          </t>
+        </section>
+        <section title="setCodecPreferences" anchor="sec.transceiver-set-codec-preferences">
+          <t>
+            The setCodecPreferences method sets the codec
+            preferences of a transceiver, which affect the
+            presence and order of codecs of an associated
+            m=section on future calls to createOffer and
+            createAnswer.
+          </t>
+          <t>
+            The codec preferences of an RtpTransceiver can
+            affect calls to createOffer and createAnswer by
```

Comment:
So 3264 has the text 

   Although the answerer MAY list the formats in their desired order of
   preference, it is RECOMMENDED that unless there is a specific reason,
   the answerer list formats in the same relative order they were
   present in the offer.  In other words, if a stream in the offer lists
   audio codecs 8, 22 and 48, in that order, and the answerer only
   supports codecs 8 and 48, it is RECOMMENDED that, if the answerer has
   no reason to change it, the ordering of codecs in the answer be 8,
   48, and not 48, 8.  This helps assure that the same codec is used in
   both directions.

Which I think supports you argument that if the JS going the create answer set the a preference, then we reorder the codecs in the offer to match that preference when generating the answer